docs/embedded-linux/configure-external-cmake-project-as-embedded-linux-project.md

```diff
@@ -0,0 +1,22 @@
+# Configure an External CMake Project as Embedded Linux Project
+
+You can now configure an external CMake project to be an Embedded Linux IoT Project, which you can develop leveraging the container toolchain: `armv7`, `arm64` or `x86`. It is convenient for cross-compilation or simulation.
+
+## Configure an External CMake Project to be an Embedded Linux Project
+
+1. Open your external CMake project folder, which is supposed to have `CMakeLists.txt` file under the root directory.
+
+2. Open command palette and select "**Azure IoT Device Workbench: Configure Project for Device Development Environment...**"
```

Comment:
It would be better to provide a simple description here to describe what this command will do behind the scene.